screenpy_selenium/resolutions/is_present.py

```diff
@@ -0,0 +1,19 @@
+"""
+Matches a present WebElement.
+"""
+
+from screenpy.resolutions.base_resolution import BaseResolution
+
+from .custom_matchers import is_present_element
+
+
+class IsPresent(BaseResolution):
```

Comment:
I worry a little that `IsPresent` might be misunderstood by someone using it to also mean that it is visible. Maybe "Exists" would be clearer? Or maybe add a note to the docstring that clearly spells out that `IsPresent` does not make any test against visibility?

To be honest, from an end-to-end "user-focused" test perspective, i'm not sure if "presence" of an element is useful. Can you think of a scenario where this kind of test would need presence rather than visibility?

Reply:
> I worry a little that IsPresent might be misunderstood by someone using it to also mean that it is visible. Maybe "Exists" would be clearer?

I thought about that too.  I decided against using a name different from `expected_conditions` module in selenium.   
`IsPresent` =~ `EC.presence_of_element_located`
Though, perhaps adding an alias would be sufficient?

Reply:
I do like the parallel. Yeah, maybe just toss an alias on it and add the note to the docstring. The generated documentation pulls from the docstring, so the note would even be on the ScreenPy Selenium API page then.

Reply:
Added `Exists` alias.

Reply:
I feel bad for this one, but... would you also be able to add an `Exist` alias for the `DoesNot(Exist())` use case?

Reply:
Added

Reply:
> Can you think of a scenario where this kind of test would need presence rather than visibility?
I use presence of element in cases where something is not enabled or displayed.  So it's in the DOM but it's not something the user can interact with.  The use cases are not large when it comes to user testing.  I've only really utilized it a couple of times directly in testing and usually the reason was purely as a means of waiting.
e.g. "wait for this easy to identify element which loads after some complicated/impossible-to-identify element"